src/mmw/apps/core/decorators.py

```diff
@@ -66,3 +67,9 @@ def _get_remote_addr(request):
             return [x.strip() for x in ipaddr.split(",")][0]
     else:
         return request.META.get("REMOTE_ADDR", "")
+
+
+def _is_api_call(request):
+    # request.auth is None for SessionAuthentication,
+    # but a Token instance for TokenAuthentication
+    return request.auth is not None
```

Comment:
This is perhaps a low-priority edge case, but I think at least worth bringing up: Making a swagger API call with an MMW session will cause a false negative (maybe because `SessionAuthentication` is listed first in the `authentication_classes` it takes precedence). I think this will be the case for all users making their API requests from the browser (eg from a notebook).

We might want to consider another method of filtering, like checking if the `TokenAuthentication` user is the `MMW_CLIENT_APP_USER`.

Reply:
Terence just pointed out to me it doesn't make any sense at all for the session info to be sent from a non-MMW app instance, so we don't have to worry about python notebooks or anything like that.

If we want to keep track of the swagger calls separately or exclude them from the UI counts, we can keep track of the referrer which will end in `/api/docs`.